pkg/controller/networkpolicy/store/addressgroup.go

```diff
@@ -162,6 +162,19 @@ func NewAddressGroupStore() storage.Interface {
 			// ag.Selector.Namespace == "" means it's a cluster scoped group, we index it as it is.
 			return []string{ag.Selector.Namespace}, nil
 		},
+		PodIndex: func(obj interface{}) ([]string, error) {
+			ag, ok := obj.(*types.AddressGroup)
+			if !ok {
+				return []string{}, nil
+			}
+			keys := make([]string, 0)
+			for _, pod := range ag.Pods {
+				name, namespace := pod.Pod.Name, pod.Pod.Namespace
+				// TODO: this feels a bit hacky, could be replaced with or more formal uid key function
+				keys = append(keys, name + "/" + namespace)
```

Comment:
It's probably fine, we do that a lot :) I don't remember if there is a "built-in" K8s function to do that